contracts/smart-account/SmartAccount.sol

```diff
@@ -17,8 +17,8 @@ import {IAuthorizationModule} from "./interfaces/IAuthorizationModule.sol";
  *         - It is modular by nature. UserOp and txns validation happens in Authorization Modules.
  *         - It provides the functionality to execute AA (EIP-4337) userOps. Gnosis style txns removed to a module.
  *         - It allows to receive and manage assets.
- *         - It is responsible for managing the modules and fallbacks.
- *         - The Smart Account can be extended with modules, such as Social Recovery, Session Key and others.
+ *         - It is responsible for managing the _modules and fallbacks.
```

Comment:
Does the linting rule also include comments?

Reply:
no no